backend/src/schema/resolvers/socialMedia.spec.js

```diff
@@ -5,17 +5,42 @@ import { host, login, gql } from '../../jest/helpers'
 const factory = Factory()
 
 describe('SocialMedia', () => {
-  let client
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-17T10:31:11Z" title="Wednesday, July 17th 2019, 12:31:11 pm +02:00">Jul 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -15,6 +15,14 @@ describe('SocialMedia', () => {
       }
     }
   `
+  const mutationU = gql`
```


I know you were probably following naming convention here, but we have talked about naming variables such that it is easy to understand what it is they are. 
In this case, I can quickly see that it is an `UpdateSocialMedia` mutation, but if I run into the variable later in the code, I might have to spend time trying to figure out what it is, you know?
Consider naming it `updateSocialMediaMutation`, or something else that makes sense.